charts/argo/Chart.yaml

```diff
@@ -2,4 +2,4 @@ apiVersion: v1
 appVersion: "v2.2.1"
 description: A Helm chart for Kubernetes
 name: argo
-version: 0.3.2
+version: 0.4.0
```

Comment:
Should this be `0.3.3`?

Reply:
I think minor bump is ok since minor typically means "add functionality in a backwards-compatible manner",and I believe this is the case. Thoughts?

Reply:
semver - you're correct.